composer.json

```
@@ -20,8 +20,8 @@
     }
   ],
   "require": {
-    "php": ">=7.1.3",
-    "illuminate/encryption": "^5.8|^6.0|^7.0"
+    "php": ">=7.3",
```

```suggestion
    "php": ">=7.1.3",
```
Bumping the php version like makes this PR a breaking change